godtools/App/Share/SwiftUI Views/SearchBar/SearchBar.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  SearchBar.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 8/29/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import SwiftUI
+
+// adapted from https://www.appcoda.com/swiftui-search-bar/
+
+struct SearchBar: View {
+    
+    @Binding var searchText: String
+    let localizationServices: LocalizationServices
```

Comment:
Here we don't want to View referencing a DataLayer class.  The View shouldn't know anything about the DataLayer.  That would violate our clean architecture.

Reply:
I hated having a whole viewModel just to translate "cancel", when this should be a built-in SwiftUI component anyway (there is one in SwiftUI, but it's not customizable), but I realize it does violate clean architecture.  Would you think I should move all the vars into a viewModel, or just localizationServices?